Microsoft.Toolkit.Uwp.SampleApp/SamplePages/PowerBIEmbedded/PowerBIEmbeddedPage.xaml.cs

```diff
@@ -0,0 +1,82 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Microsoft.Toolkit.Uwp.SampleApp.SamplePages
+{
+    public sealed partial class PowerBIEmbeddedPage
+    {
+        public PowerBIEmbeddedPage()
+        {
+            this.InitializeComponent();
+        }
+
+        private void PropertyDropdown_SelectionChanged(object sender, SelectionChangedEventArgs e)
+        {
+            PropertyValue.Text = string.Empty;
+
+            switch (PropertyDropdown.SelectedIndex)
+            {
+                case 0:
```

Comment:
Would be nice to see binding here instead?

Reply:
@michael-hawker Done, data trigger isn't supported in UWP and no compare converter defined in this solution, so I use behavior instead